src/components/text-editor/examples/text-editor-with-tables.tsx

```diff
@@ -26,6 +26,7 @@ export class TextEditorWithTablesExample {
                 onChange={this.handleChange}
                 readonly={this.readonly}
                 contentType="html"
+                enableTables={true}
```

Comment:
Unless I've missed something, it seems like this last commit (`fix: configure table handling`) should just be a part of the preceding commit, rather than a commit of its own?

Reply:
It should, but I didn't want to squash in case it's preferred to put the configuration in a different way

Reply:
Alright, I'll change it to a fixup commit then 👍